mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -354,6 +358,18 @@ impl RpcCommonOps for TendermintCoin {
     }
 }
 
+#[derive(PartialEq)]
+pub enum TendermintWalletConnectionType {
+    Wc,
+    WcLedger,
+    KeplrLedger,
+    Internal,
```

Comment:
This is missing Keplr mode and I am not sure what `Internal` refers to, I guess `Native`?

Reply:
yeah, renamed `Internal` to `Native`